Pear/Cells/FeedbackMenuTableViewCell.swift

```diff
@@ -0,0 +1,42 @@
+
+//
+//  FeedbackMenuTableViewCell.swift
+//  Pear
+//
+//  Created by Amy Chin Siu Huang on 3/21/21.
+//  Copyright © 2021 cuappdev. All rights reserved.
+//
+import UIKit
+
+class FeedbackMenuTableViewCell: UITableViewCell {
```

Comment:
I'm wondering if we technically need a cell file just for the `FeedbackMenuTableViewCell` because it is quite basic and we will never have more than 3 of these cells so reusing cells doesn't seem very important. I'm wondering if it might simplify our code a little bit if you just directly returned the cell in the `FeedbackView.swift` file?